air/src/execution_step/instructions/new.rs

```diff
@@ -51,7 +51,9 @@ fn prolog<'i>(new: &New<'i>, exec_ctx: &mut ExecutionCtx<'i>) {
     match &new.argument {
         NewArgument::Stream(stream) => {
             let iteration = exec_ctx.tracker.new_tracker.get_iteration(position);
-            exec_ctx.streams.meet_scope_start(stream.name, new.span, iteration);
+            exec_ctx
+                .streams
+                .meet_scope_start(stream.name, new.span, iteration as usize);
```

Comment:
What do you think about `try_into()` too, or `static_assert!(size_of(u32) <= size_of(usize));`?

We had similar at #544.

Reply:
yep, I just missed it